src/main/java/athena/task/Time.java

```diff
@@ -38,6 +38,7 @@ public Time(Boolean isFlexible, String startTime, String duration, String deadli
         this.recurrence = recurrence;
 
         setRecurrence(recurrence);
+        assert recurrenceDates.size() != 0;
```

Comment:
Strictly speaking this shouldn't be part of this PR. But this is good.

Reply:
Okay I'll delete this and add it in another branch